SPECS/gdb/gdb.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        C debugger
 Name:           gdb
-Version:        11.2
+Version:        13.2
```

Comment:
@AZaugg there seems to be a `merge` conflict.

Reply:
Triggered a new test build: [PR-6034](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=441798&view=results)

Reply:
Test build ✅ 
/cc @anphel31